src/replica/ConfigParserMySQL.cc

```diff
@@ -92,7 +95,7 @@ void ConfigParserMySQL::_parseWorkers() {
 }
 
 void ConfigParserMySQL::_parseDatabaseFamilies() {
-    _conn->execute("SELECT * FROM " + _conn->sqlId("config_database_family"));
+    _conn->execute(_g.select(Function::STAR) + _g.from("config_database_family"));
```

Comment:
I am not sure that any of these two are better than simple `SELECT * FROM config_database_family`?

Reply:
The select is the variadic function templates that accept parameters of various types, including values (of the string, numeric, etc.), identifiers (columns), functions, etc. Some need to be processed by quoting them using appropriate quotes (identifiers need to be quoted with back ticks, strings - with the single quotes). Others (reserved SQL words, functions) should not be. For example, the following:
```
_g.select(Sql::STAR, "abc", 123, _g.id("obejctId"), Sql::NOW)
```
will result in:
```
SELECT *, 'abc', 123, `objectId`, NOW()
```
If `*` would be passed as the plan string then it would result in treating it as the string value and produce the incorrect output:
```
SELECT '*', 'abc', 123, `objectId`, NOW()
```
That's the whole point of introducing `Sql::STAR` (and `Sql::COUNT_STAR`) here.